app/src/main/java/org/wikipedia/edits/EditsSummary.kt

```diff
@@ -1,11 +1,11 @@
-package org.wikipedia.suggestededits
+package org.wikipedia.edits
 
 import org.wikipedia.Constants
 import org.wikipedia.gallery.ExtMetadata
 import org.wikipedia.page.PageTitle
 import org.wikipedia.util.ImageUrlUtil
 
-data class SuggestedEditsSummary(
+data class EditsSummary(
```

Comment:
`EditsSummary` sounds very odd. Can we come up with a more meaningful name? If someone looks at this code for the first time, would they know what this is for?  Perhaps something like `PageSummaryForEdit`?